tests/files/environments/development/us-east-1.yml

```diff
@@ -0,0 +1,15 @@
+defaults:
+  parameters:
+    CidrBlock: '10.0'
+stacks:
+  - name: Test-Dev-VPC
+    template_name: VPC
+    parameters:
+      Name: "TestVPC"
+
+  - name: Test-Dev-SecurityGroup
+    template_name: SecurityGroup
+    parameters:
+      VpcId:
+        Stack: Test-VPC
+        Output: VpcId
```

Comment:
Could even store the region-specific parameter overrides in the environment file, and remove the `stacks` directory: 

```
environments/sandbox.yml # parameters, default region, no stacks — environment is inferred by existence of file
environments/sandbox/SecurityGroup.yml # can override region here, if different from environment
```

Just seems like this makes it easier to see all the stacks and modify them using the filesystem structure.